test/unit/check_modules_test.js

```diff
@@ -0,0 +1,60 @@
+/* global describe, it */
+
+var path = require('path')
+
+var expect = require('chai').expect
+var checkModules = require('../../lib/check_modules')
+
+describe('Checking modules', function () {
+  var fixture_path = path.resolve(__dirname, '..', 'fixtures')
+
+  describe('when they are valid', function () {
+    it('does not throw an exception', function () {
+      var packageInfo = require('../fixtures/valid_dependencies/package.json')
```

Comment:
Perhaps this (and the other two declarations in the other two tests) can be moved to directly under the `describe('Checking modules' ...` block?